packages/fxa-settings/tailwind.config.js

```diff
@@ -0,0 +1,181 @@
+module.exports = {
+  purge: {
+    content: ['./src/**/*.tsx', './public/index.html'],
+    whitelist: [],
+  },
+  theme: {
+    screens: {
+      sm: '480px',
+      md: '768px',
+      lg: '1024px',
+      xl: '1441px',
+    },
+    colors: {
+      black: '#000',
+      white: '#fff',
+      link: {
+        default: '#0060df',
+        hover: '#0250bb',
+        visited: '#b833e1',
+        'hover-visited': '#952bb9',
+      },
+      mozilla: {
+        'neon-blue': '#00ffff',
+        'lemon-yellow': '#fff44f',
+        'warm-red': '#ff4f5e',
+        'neon-green': '#54ffbd',
+        'dark-purple': '#6e008b',
+        'dark-green': '#005e5e',
+        'dark-blue': '#00458b',
+        'dark-gray': '#959595',
+        'light-gray': '#e7e5e2',
+      },
+      green: {
+        50: '#e3fff3',
+        100: '#d1ffee',
+        200: '#b3ffe3',
+        300: '#88ffd1',
+        400: '#54ffbd',
+        500: '#3fe1b0',
+        600: '#2ac3a2',
+        700: '#008787',
+        800: '#005e5e',
+        900: '#08403f',
+      },
+      blue: {
+        50: '#aaf2ff',
+        100: '#80ebff',
+        200: '#00ddff',
+        300: '#00b3f4',
+        400: '#0090ed',
+        500: '#0060df',
+        600: '#0250bb',
+        700: '#054096',
+        800: '#073072',
+        900: '#09204d',
+      },
+      violet: {
+        50: '#e7dfff',
+        100: '#d9bfff',
+        200: '#cb9eff',
+        300: '#c689ff',
+        400: '#ab71ff',
+        500: '#9059ff',
+        600: '#7542e5',
+        700: '#592acb',
+        800: '#45278d',
+        900: '#321c64',
+      },
+      purple: {
+        50: '#f7e2ff',
+        100: '#f6b8ff',
+        200: '#f68fff',
+        300: '#f770ff',
+        400: '#d74cf0',
+        500: '#b833e1',
+        600: '#952bb9',
+        700: '#722291',
+        800: '#4e1a69',
+        900: '#2b1141',
+      },
+      pink: {
+        50: '#ffdef0',
+        100: '#ffb4db',
+        200: '#ff8ac5',
+        300: '#ff6bba',
+        400: '#ff4aa2',
+        500: '#ff298a',
+        600: '#e31587',
+        700: '#c60084',
+        800: '#7f145b',
+        900: '#50134b',
+      },
+      red: {
+        50: '#ffdfe7',
+        100: '#ffbdc5',
+        200: '#ff9aa2',
+        300: '#ff848b',
+        400: '#ff6a75',
+        500: '#ff4f5e',
+        600: '#e22850',
+        700: '#c50042',
+        800: '#810220',
+        900: '#440306',
+      },
+      orange: {
+        50: '#fff4de',
+        100: '#ffd5b2',
+        200: '#ffb587',
+        300: '#ffa266',
+        400: '#ff8a50',
+        500: '#ff7139',
+        600: '#e25920',
+        700: '#cc3d00',
+        800: '#9e280b',
+        900: '#7c1504',
+      },
+      yellow: {
+        50: '#ffffcc',
+        100: '#ffff98',
+        200: '#ffea80',
+        300: '#ffd567',
+        400: '#ffbd4f',
+        500: '#ffa436',
+        600: '#e27f2e',
+        700: '#c45a27',
+        800: '#a7341f',
+        900: '#960e18',
```

Comment:
Do we want to go ahead and include all of these colors off the bat when we likely won't be using them, or should we start with what we need (blues, greys, whatever else)? I can see both sides - it doesn't hurt anything since we've got purgeCSS and this aligns us to colors from Protocol which Phil said we'd be using, but it's also some extra clutter and length in this file... what do you think? I like the `color-number` approach though.

Reply:
I take your point, my personal preference is to keep them in, simply because they're not adding any extra bulk to the output and if we do end up needing to use a color it'll immediately be available / no out of scope diff in the commit. If it's a file length thing I would be cool moving it to a shared JSON file that we import? I can be swayed to remove them though if there are strong opinions!

Reply:
I think one other thing that could come out of listing _all_ of the values (of blues for example) could be someone accidentally using `blue-300` when they should have used `blue-400`, when `blue-50` through `blue-300` shouldn't be used anywhere in the project. It could open up the possibility of using one assumed to be the right one because just because it's there, but that could also happen with someone using `blue-500` when they should have used `blue-400`. I guess ultimately it's up to dev/UX to catch it so I'm kind of torn.

I guess we can't sync up these values to protocol's package because we don't want that as a dependency, but what if their colors change in the future? We could have a bunch of colors we're not even using that would be inaccurate. Maybe that's not a concern though because that would only happen with rebranding?

Maybe a happy medium is pulling out values we _really_ don't think will be used, but even then I feel like there are quite a few. I don't see us ever using `violet`, `pink`, `purple`, `ink`, possibly `marketing-gray`... If we want our styles to be really consistent, maybe we should only list the ones we're actually going to use in the design, add others as needed, and keep the others somewhere else easily accessible when we do need them since you've already done the work for it. They could be listed when we get around to a design or branding doc. I guess ultimately I'm OK with these living in another file we pull in, but I'm not convinced we're _ever_ going to use some of them in FxA so I'm not sure we need them now because they could change.

Reply:
I'm fine with having them all here from the start. I'd rather have the snapshot of all the Protocol colors than bring them in as needed because it's one less thing to change during regular dev and color updates will prob be easier to diff when done all at once.

Reply:
@dannycoates Jody and I had a meeting with Phil and Jess earlier, they have a different set of colors for us to use that aren't exactly Protocol colors. Not all of them will be used with the current design but we were told those colors will be the bucket we pull from in the future when we do need to use those shades so we're all on the same page. 👍